routes.go

```diff
@@ -307,6 +311,75 @@ func (s *Server) publishEvent(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// getCurrentState get current status of the  events that are subscribed to
```

Comment:
Do we have to document other return codes for this API route.

Reply:
add comment for swagger